mmv1/templates/terraform/examples/cloudbuild_trigger_allow_failure.tf.erb

```diff
@@ -0,0 +1,66 @@
+resource "google_cloudbuild_trigger" "<%= ctx[:primary_resource_id] %>" {
+  location = "global"
+
+  trigger_template {
+    branch_name = "main"
+    repo_name   = "my-repo"
+  }
+
+  build {
+    step {
+      name = "ubuntu"
+      args = ["-c", "exit 1"]
+      allowFailure = true # using allowFailure field
```

Comment:
It should be `allow_failure = true` and the comment for this field is not necessary.